examples/yoti_example_django/yoti_example/views.py

```diff
@@ -1,51 +1,80 @@
 from django.views.generic import TemplateView
 from dotenv import load_dotenv, find_dotenv
 
-load_dotenv(find_dotenv())
 
 from yoti_python_sdk import Client
+from yoti_python_sdk.dynamic_sharing_service import (
+    DynamicScenarioBuilder,
+    create_share_url,
+)
+from yoti_python_sdk.dynamic_sharing_service.policy import DynamicPolicyBuilder
+
+load_dotenv(find_dotenv())
+
 from app_settings import (
     YOTI_SCENARIO_ID,
     YOTI_CLIENT_SDK_ID,
-    YOTI_KEY_FILE_PATH
-)
+    YOTI_KEY_FILE_PATH,
+)  # noqa
```

Comment:
loaddotenv being before it/it being a module level import that's not at the start of the file. Because of how app_settings is written, it sets the values of these constants to values loaded in by dotenv.